cli/src/semgrep/run_scan.py

```diff
@@ -312,9 +312,9 @@ def run_rules(
 def list_targets_and_exit(target_manager: TargetManager, product: out.Product) -> None:
     targets = target_manager.get_files_for_language(None, product)
     for path in sorted(targets.kept):
-        print(f"+ {path}")
+        print(f"SEL {path}")
```

Comment:
nit: SEL + IGN feels less clear to me  in meaning as a developer than + or -. Maybe we should just say "selected" and "ignored", or keep the + and -

Reply:
Sure. `+` and `-` make it hard to read the diffs that add their own layer of `+` and `-`.